src/main/java/au/org/ala/names/index/NameProvider.java

```diff
@@ -312,6 +316,24 @@ public String getUnknownTaxonID() throws IndexBuilderException {
         return utid;
     }
 
+    /**
+     * Get the name of the default parent taxon.
+     * <p>
+     * If, during resolution, a taxon from this provider does not have a parent, then a reference
+     * to this name (and default nomenclatural code, if one is available) is used.
+     * If one is not explicitly set, get the parent.
+     * </p>
+     *
+     * @return The name of the parent taxon, or null for none.
+     *
+     */
+    @Nullable
+    public String getDefaultParentTaxon() {
+        if (this.defaultParentTaxon == null && this.parent != null)
```

Comment:
Just a reminder for next time - ALA coding style prefers braces to always be used for if statements.